src/Easydropdown/Easydropdown.test.ts

```diff
@@ -152,4 +152,30 @@ describe('Easydropdown', () => {
             assert.equal(edd.dom.item.length, 7);
         });
     });
+
+    describe('.validate()', () => { 
+
+        it('validates a required select', () => {
+            const select = createSelect('required');
+            const edd = new Easydropdown(select, {});
+
+            // @ts-ignore
+            assert.equal(edd.validate(), false);
+            edd.actions.selectOption(1);
+            // @ts-ignore
+            assert.equal(edd.validate(), true);
+        });
+
+        it('validates a non required select', () => {
+            const select = createSelect();
+            const edd = new Easydropdown(select, {});
+
+            // @ts-ignore
+            assert.equal(edd.validate(), true);
+            edd.actions.selectOption(1);
+            // @ts-ignore
+            assert.equal(edd.validate(), true);    
+        });    
```

Comment:
My linter is saying you have some trailing whitespace around these blocks - are you able remove it? (vscode has an option to auto remove on save if you're using vscode)

Reply:
enabled it but cant see any differnce. would be nice if you could recheck.

Reply:
Yeah all looks good, thanks!